src/AutomatedCar/SystemComponents/Packets/IPowerTrainPacket.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace AutomatedCar.SystemComponents.Packets
+{
+    public enum GearShifterPosition
```

Comment:
Miért van itt is egy ilyen enum, amikor tettél egy ugyanilyet a GearShifter-be is?

Reply:
Eleinte itt volt, de ezt innen kiszedtem (legalabbis ki akartam), miutan atraktam a GearShifter-be... valamiert ez itt nem jott meg at, javitom rovidesen

Reply:
...cserebe igy a buildnel hibat dob a git, mert egyelore masteren nincs definialva a GearShifterPosition, mert az meg szinten kulon feature branchen van, a GearShifterben. Ezt hogy lehetne szepen megoldani?